docs/README.md

```diff
@@ -11,6 +11,8 @@ INSTALL open_prompt FROM community;
 LOAD open_prompt;
 ```
 
+Please make sure you are using DuckDB version 1.1.2 or higher.
```

Comment:
Thanks for adding the specification. Since this is a requirement it probably sits best in the 'Requirements' block?

Reply:
If you put the installation before the requirements, I would put it in the installation section, otherwise you find that it doesn't work before you read the requirements :)

Reply:
I see there is no requirements section. Okay, if you create it, I would include it before installation